src/runtime/importhook.cs

```diff
@@ -37,6 +36,11 @@ class DotNetFinder(importlib.abc.MetaPathFinder):
         if 'clr' not in sys.modules:
             return None
         clr = sys.modules['clr']
+
+        _add_pending = clr._add_pending_namespaces
+        if _add_pending:
```

Comment:
Why are you checking it instead of calling it directly?

Reply:
I wasn't sure if there are no scenarios where clr module won't have its members loaded yet, but would already be present in `sys.modules`. The uncertainty was based on the lines that immediately follow this one. I guess I could skip the check, and reintroduce it if some scenario is reported.